packages/connect-node/src/http2-session-manager.ts

```diff
@@ -305,11 +305,7 @@ export class Http2SessionManager {
       case "verifying":
         state.verified.then(
           (value) => {
-            if ("t" in value) {
-              this.setState(value);
-            } else {
-              this.setState(ready(value, this.options));
-            }
+            this.setState(value);
```

Comment:
Unrelated cleanup. This was a remnant from earlier in the implementation, when the promise resolved to a state or to a http2 connection that is no longer necessary. The else block is dead code.